--- a/arch/arm/dts/k3-j721e-common-proc-board-u-boot.dtsi	2024-03-27 15:38:28.317898700 +0900
+++ b/arch/arm/dts/k3-j721e-common-proc-board-u-boot.dtsi	2024-03-19 16:19:39.325169400 +0900
@@ -181,9 +181,11 @@
 	bootph-pre-ram;
 };
 
+/*
 &exp2 {
 	bootph-pre-ram;
 };
+*/
 
 &mcu_fss0_ospi0_pins_default {
 	bootph-pre-ram;
@@ -271,6 +273,7 @@
 	/delete-property/ assigned-clock-parents;
 };
 
+/*
 &serdes0_pcie_link {
 	assigned-clocks = <&serdes0 CDNS_SIERRA_PLL_CMNLC>;
 	assigned-clock-parents = <&wiz0_pll1_refclk>;
@@ -280,3 +283,4 @@
 	assigned-clocks = <&serdes0 CDNS_SIERRA_PLL_CMNLC1>;
 	assigned-clock-parents = <&wiz0_pll1_refclk>;
 };
+*/
