--- arch/arm64/boot/dts/ti/k3-j721e-main.dtsi	2025-02-04 11:26:37.333381880 +0900
+++ arch/arm64/boot/dts/ti/k3-j721e-main.dtsi	2025-02-05 15:44:36.548862121 +0900
@@ -1623,8 +1623,10 @@
 		bus-width = <8>;
 		mmc-hs200-1_8v;
 		mmc-ddr-1_8v;
-		ti,otap-del-sel-legacy = <0x0>;
-		ti,otap-del-sel-mmc-hs = <0x0>;
+		//ti,otap-del-sel-legacy = <0x0>;
+		//ti,otap-del-sel-mmc-hs = <0x0>;
+		ti,otap-del-sel-legacy = <0xf>;
+		ti,otap-del-sel-mmc-hs = <0xf>;
 		ti,otap-del-sel-ddr52 = <0x5>;
 		ti,otap-del-sel-hs200 = <0x6>;
 		ti,otap-del-sel-hs400 = <0x0>;
@@ -1632,8 +1634,10 @@
 		ti,itap-del-sel-mmc-hs = <0xa>;
 		ti,itap-del-sel-ddr52 = <0x3>;
 		ti,trm-icp = <0x8>;
+		ti,strobe-sel = <0x77>;
 		dma-coherent;
-		status = "disabled";
+		// status = "disabled";
+		status = "okay";
 	};
 
 	main_sdhci1: mmc@4fb0000 {
@@ -1661,7 +1665,8 @@
 		ti,clkbuf-sel = <0x7>;
 		dma-coherent;
 		sdhci-caps-mask = <0x2 0x0>;
-		status = "disabled";
+		// status = "disabled";
+		status = "okay";
 	};
 
 	main_sdhci2: mmc@4f98000 {
