--- arch/arm64/boot/dts/ti/k3-j721e-common-proc-board.dts	2024-08-07 19:22:09.090978185 +0900
+++ arch/arm64/boot/dts/ti/k3-j721e-common-proc-board.dts	2024-10-29 09:55:17.381602977 +0900
@@ -575,23 +575,32 @@
 };
 
 &wkup_i2c0 {
+	status = "disabled";
+	/*
 	status = "okay";
 	pinctrl-names = "default";
 	pinctrl-0 = <&wkup_i2c0_pins_default>;
+	*/
 };
 
 &main_i2c0 {
+	status = "disabled";
+	/*
 	status = "okay";
 	pinctrl-names = "default";
 	pinctrl-0 = <&main_i2c0_pins_default>;
 	clock-frequency = <400000>;
+	*/
 };
 
 &main_i2c1 {
+	status = "disabled";
+	/*
 	status = "okay";
 	pinctrl-names = "default";
 	pinctrl-0 = <&main_i2c1_pins_default>;
 	clock-frequency = <400000>;
+	*/
 };
 
 &main_i2c2 {
