--- arch/arm64/boot/dts/ti/k3-j721e-common-proc-board.dts	2024-10-29 14:47:32.431031500 +0900
+++ arch/arm64/boot/dts/ti/k3-j721e-common-proc-board.dts	2024-10-30 13:48:14.971715405 +0900
@@ -319,6 +319,13 @@
 };
 
 &wkup_pmx0 {
+
+	wkup_gpio_pins_default: wkup-gpio-pins-default {
+		pinctrl-single,pins = <
+			J721E_WKUP_IOPAD(0xD0, PIN_OUTPUT, 7) /* (G27) WKUP_GPIO0_8 */ 
+		>;
+	};
+
 	mcu_fss0_ospi0_pins_default: mcu-fss0-ospi0-pins-default {
 		pinctrl-single,pins = <
 			J721E_WKUP_IOPAD(0x0000, PIN_OUTPUT, 0) /* MCU_OSPI0_CLK */
@@ -413,6 +420,21 @@
 	status = "disabled";
 };
 
+&wkup_gpio0 {
+	// status = "disabled";
+	pinctrl-names = "default";
+	pinctrl-0 = <&wkup_gpio_pins_default>;
+
+	// TC397_AP1_GPIO_TEST5
+	mcu_ap1_gpio_test5 {
+		/* G27 - WKUP_GPIO0_8 */
+		gpio-hog;
+		gpios = <8 GPIO_ACTIVE_HIGH>;
+		output-high;
+		line-name = "GPIO_TEST5";
+	};
+};
+
 &wkup_gpio1 {
 	status = "disabled";
 };
